app/grants/clr.py

```diff
@@ -35,6 +35,12 @@
 PREV_CLR_START_DATE = dt.datetime(2020, 3, 23, 12, 0)
 PREV_CLR_END_DATE = dt.datetime(2020, 4, 7, 12, 0)
 CLR_START_DATE = dt.datetime(2020, 6, 15, 12, 0)
+CLR_END_DATE = dt.datetime(2020, 7, 3, 16, 0)
+
+try:
+    CLR_END_DATE
+except NameError:
```

Comment:
what is this for

Reply:
Just to be extra safe to ensure no one comments out `CLR_END_DATE` while round is active 
That check ensure if that variable is not defined -> set it to current timezone